checkov/serverless/utils.py

```diff
@@ -54,7 +54,7 @@ def get_scannable_file_paths(root_folder: str | None = None, excluded_paths: lis
         return files_list
 
     for root, d_names, f_names in os.walk(root_folder):
-        # Don't walk in to "node_modules" directories under the root folder. If –for some reason–
+        # Don't walk in to "node_modules" directories under the root folder. If for some reason
```

Comment:
This line was changed by the script I made to go over this repo to find bad unicode characters.
I left it in the PR just because unicode seems unnecessary here. There's no bug in this line.